app/src/main/res/values/styles_v2.xml

```diff
@@ -154,6 +154,11 @@
     <item name="android:paddingBottom">18dp</item>
   </style>
 
+  <style name="Clinic.V2.RoundedClearButton">
```

Comment:
`RoundedClearButton` -> `RoundedClearEditTextButton`? `RoundedClearFieldButton`? `RoundedClearInputButton`?

Reply:
Ah, the name is unclear. by clear, I mean the background colour is translucent. The naming is wrong, I'll improve it.

FYI, this is the background for the buttons like "Not You?" on the app lock screen and the "Edit" button on the patient summary screen.

Reply:
Clearly 😄

Reply:
CLEARly 😄